docs/src/operate/registration/register.md

```diff
@@ -86,18 +86,22 @@ GET /v1/entities/{topic-id}
 curl http://localhost:8000/tedge/entity-store/v1/entities/device/child01
 ```
 
-## Query Entities
-
-### Query All Entities
-
-List all entities registered with thin-edge starting from the `main` device.
+## Query entities
 
 **Endpoint**
 
 ```
 GET /v1/entities
 ```
 
+**Query parameters**
+
+| Parameter | Description                                                  | Examples                            |
+|-----------|--------------------------------------------------------------|-------------------------------------|
+| `root`    | The topic id of the root node where to start the search from | `device/child2//`                   |
```

Comment:
Might be worthwhile mentioned that using the `root` query parameter includes the root element.

"The topic id of the root node where to start the search from (including the root entity)"

Reply:
Resolved by https://github.com/thin-edge/thin-edge.io/commit/1de9671aaddddc34a5e2b00b8d3076a72018064b